package.json

```diff
@@ -23,6 +23,7 @@
     "grunt-contrib-jshint": "0.11.0",
     "grunt-jscs": "1.5.0",
     "grunt-mocha-cli": "1.12.0",
+    "istanbul": "^0.4.3",
```

Comment:
I was gonna wait till the eslint + newest coinsvalidate, not sure actually why it was installed (I didnt add it)... I think it was coinsvalidate's fault as well... :-P